pytoda/datasets/annotated_dataset.py

```diff
@@ -0,0 +1,92 @@
+"""Implementation of AnnotatedDataset class."""
+import torch
+import pandas as pd
+from torch.utils.data import Dataset
+from ..types import DrugSensitivityData
+
+
+class AnnotatedDataset(Dataset):
+    """
+    Annotated dataset implementation.
+    """
+
+    def __init__(
+        self,
+        annotations_filepath: str,
+        input_data: Dataset,
+        index_col: int,
+        device: torch.device = torch.
+        device('cuda' if torch.cuda.is_available() else 'cpu'),
+        backend: str = 'eager',
+        **kwargs
+    ) -> None:
+        """
+        Initialize an annotated dataset.
+        E.g. the  input_data could be SMILES and the annotations could be
+        single or multi task labels.
+
+        Args:
```

Comment:
Updated the docstring according to the arguments. In case we "pop" index_col from kwargs add the fact that this argument should not be provided in the docstring.

Reply:
should also be fixed with latest commit